code/game/objects/items/stacks/medical.dm

```diff
@@ -198,8 +198,8 @@
 					continue
 				if (W.bandaged && W.disinfected)
 					continue
-				if(used == amount)
-					break
+				//if(used == amount) //VOREStation Edit
+				//	break //VOREStation Edit

```

Comment:
Changed ATKs but not gauze?

Reply:
Correct. My thought is to give ATKs a bit of a usability boost over standard gauze.